### ISSUE TITLE
All pages: Instruction for expand/collapse sub-menu is not clear

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#32]
URL: https://hcdev.openplus.ca/en/user/
User Agent: Chrome with NVDA
Language: English/French

Issue Description: The instruction to open/close the sub-menu tray is not clear to the user. It reads " toggle button not pressed My account". It has the same state "not pressed" when it's selected or not. Also, when the user selects it, it announces "tray closed" or "tray opened". This is not enough instruction to a non-sighted user to what has happened.
Please see screenshot with NVDA speech viewer on the side.

Steps to reproduce: Open NVDA and either click with mouse or select with keyboard.

Expected results: It should announce the correct state (either selected or not selected) and it should tell the user what will happen when they click it (eg. selecting will open a sub menu or hide it).

![image](https://user-images.githubusercontent.com/14280062/128216733-705c2f5b-997e-4c1c-9dfd-5fa46c6db16d.png)

**Comment by echouin** [comment]
This is a Drupal Core issue and is out of scope for OP.
From OP:
After investigation, found that this problem derived from the core toolbar module. I did not need to open a Drupal.org ticket because this has already been flagged as major in the issue queue.
﻿﻿https://www.drupal.org/project/drupal/issues/3085781﻿
﻿
﻿There is also a larger thread focused on accessibility that may be of interest.
﻿
﻿﻿https://www.drupal.org/project/drupal/issues/3084529[](javascript:void(0))
